js/index.js

```diff
@@ -276,3 +281,9 @@ function showAlert(message) {
     }, 5000);
 }
 
+function showSuccess(message) {
```

Comment:
Although I would like to change the showAlert to showFailAlert, but it seems a great work